testing/TestHarness/TestHarness.Shared/Ext/Navigation/PageNavigation/PageNavigationOnePage.xaml

```diff
@@ -35,5 +37,24 @@
 					Content="Settings Write Test"
 					Click="{x:Bind ViewModel.SettingsWriteTest}" />
 		</StackPanel>
+		<ScrollViewer Grid.Row="2">
+			<!-- Option 1 - Navigation.Request can be specified on the ItemsRepeater itself -->
+			<muxc:ItemsRepeater ItemsSource="{Binding Items}"
+								uen:Navigation.Request="PageNavigationTwo">
+				<muxc:ItemsRepeater.ItemTemplate>
+					<DataTemplate>
+						<!-- Option 2 - Navigation.Request can be specified along with the Navigation.Data attribute on the item -->
+						<Border Height="200"
+								Width="200"
+								Background="Red"
+								uen:Navigation.Data="{Binding}">
```

Comment:
I don't get the need for this binding ... just like an `ListView` the `Data` should be provided automatically using the "clicked item".